fab/inventory/production

```diff
@@ -60,5 +62,5 @@ datavol_device=/dev/xvdb
 [stanchion]
 hqriak0.internal-va.commcarehq.org
 
-[rackspace_cloud_backup]
-hqdb0.internal.commcarehq.org
+# [rackspace_cloud_backup]
+# hqdb0.internal-va.commcarehq.org
```

Comment:
What's the plan for reinstating this? (Do we plan to reinstate this? What is this again?)